pages/doc/logging_overview.md

```diff
@@ -34,7 +34,7 @@ summary: Learn about Tanzu Observability metrics, logs, and traces.
 </table>
 
 
-{%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings. To keep log data for longer, contact [technical support](wavefront_support_feedback.html#support)."%}
+{%include note.html content="Tanzu Observability retains logs for 7, 15, or 30 days, based on your settings. For longer retention needs, contact [technical support](wavefront_support_feedback.html#support)."%}
```

Comment:
I recommend removing the link to contact technical support. It's not possible to retain logs for longer than 30 days.

Reply:
I will be keeping it as discussed :) There is JIRA to track this.